packages/create-react-app/index.js

```diff
@@ -152,10 +152,10 @@ function getInstallPackage(version) {
 
 // Extract package name from tarball url or path.
 function getPackageName(installPackage) {
-  if (~installPackage.indexOf('.tgz')) {
+  if (installPackage.indexOf('.tgz') > 0) {
```

Comment:
Only reason I used `> 0` here is because ".tgz" would be an invalid name that should not be split. But that's an edge case - cool with changing to `> -1`.